lib/check_rr_enablement.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python3
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+import subprocess
+import base64
+import json
+import sys
+from kubernetes import client, config
+
+CUSTOMIZATIONS="/tmp/customization.yaml"
+
+def get_ceph_details():
+    host = 'ncn-m001'
+    cmd = f"ssh {host} 'ceph osd tree -f json-pretty'"
```

Comment:
Was this ever tested during fresh install? The `install.sh` from which this Python script is called is executed on `pit`. When system is in fresh install mode, `ncn-m001` does not exist yet. `Pit` is converted to `ncn-m001` at later stage.